includes/embeds/class-amp-youtube-embed-handler.php

```diff
@@ -151,10 +230,74 @@ public function render( $args, $url ) {
 				'width'       => $this->args['width'],
 				'height'      => $this->args['height'],
 				'placeholder' => '',
+				'start'       => 0,
 			]
 		);
 
-		if ( empty( $args['video_id'] ) ) {
+		$attributes = array_merge(
+			[ 'data-videoid' => $args['video_id'] ],
+			wp_array_slice_assoc( $args, [ 'layout', 'width', 'height' ] )
+		);
+		if ( ! empty( $args['title'] ) ) {
+			$attributes['title'] = $args['title'];
+		}
+
+		$allowed_data_params = [
+			'cc_lang_pref',
+			'cc_load_policy',
+			'color',
+			'controls',
+			'disablekb',
+			'enablejsapi',
+			'end',
+			'fs',
+			'hl',
+			'iv_load_policy',
+			'list',
+			'listType',
+			'modestbranding',
+			'origin',
+			'playlist',
+			'playsinline',
+			'rel',
+			'widget_referrer',
+		];
```

Comment:
I'm not sure if we want to hardcode all the allowed data params here since we don't control whenever they become deprecated or new ones are added. Instead, we could allow all `data-param-*` to pass through but sanitize their values.